keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -153,6 +154,10 @@ Note that some features are plain unsupported while some will not work without u
 - Not all features of shared namespaces are supported. This is clarified in the API section below
 - The existing node problem detector is Linux-only and requires privileged containers. In general, we don't expect these to be used on Windows because there's no privileged support
 - Overlay networking support in Windows Server 1803 is not fully functional using the `win-overlay` CNI plugin. Specifically service IPs do not work on Windows nodes. This is currently specific to `win-overlay`; other CNI plugins (OVS, AzureCNI) work. Since Windows Server 1803 is not supported for GA, this is mostly not applicable. We left it here since it impacts beta
+- Outbound communication using the ICMP protocol via the `win-overlay`, `win-bridge`, and `Azure-CNI` plugin. Specifically, the Windows data plane ([VFP](https://www.microsoft.com/en-us/research/project/azure-virtual-filtering-platform/)) doesn't support ICMP packet transpositions. This means:
+    - ICMP packets directed to destinations within the same network (e.g. pod to pod communication via ping) will work as expected and without any limitations
```

Comment:
Wow. I don't think we document that ICMP should work anywhere. I searched kubernetes.io docs and didn't find it.

Do we have tests for ICMP?

@thockin @caseydavenport

Reply:
I don't think we ever specified it.  pod-to-pod ping is really useful for debugging only, but things like ICMP reject are pretty important between nodes or for services with no endpoints.

Ping to the outside world is something I use for debug.  I  honestly have no idea if breaking this is going to be a big deal.  I assume the same breakage holds for ICMP reject or other control signals?  That's a pretty weird thing to break, but I am not sure it is a critical gap.

Can we bring this up at sig-net next week, or can you start an email thread on sig-net?

Reply:
Generally speaking, you can replace `ping <destination>` with `curl <destination>` to be able to debug connectivity to the outside world. Yes, I can bring it up.

Reply:
@daschott can you please add this to the docs as well that the alternative is curl/powershell to test connectivity. i will remove the hold then.

Reply:
I thought CNI provides IP.  ICMP is IP, right?